Example reorg (#29)
watts-dev/watts: examples/MultiStep_Griffin-BISON-Sockeye_MR/README.md

# MultiStep_Griffin-BISON-Sockeye_MR

## Purpose

> This example provides a demonstration on how to use WATTS to model multi-step Workflows using MOOSE codes.

Reviewer: ```suggestion
This example provides a demonstration on how to use WATTS to model multi-step Workflows for multiphysics simulation involving various MOOSE applications coupled by MOOSE's MultiApps system.
```

# MultiStep_Griffin-BISON-Sockeye_MR

## Purpose

This example provides a demonstration on how to use WATTS to model multi-step Workflows for multiphysics simulation involving various MOOSE applications coupled by MOOSE's MultiApps system.

## Code(s)
 
- MOOSE MultiApp
- Sockeye
- BISON
- Griffin

## Keywords
 
- Multi-steps
- MultiApp
- Micro-reactor
- Steady-state
- Transiens

## File descriptions

> 

Author: @miaoyinb - please review carefully this README. For instance, I am not sure why we need two mesh files in this repo, and I cannot find the ISOXML file.

Reviewer: BISON and Griffin use different mesh. Griffin cannot handle voids in mesh but BISON needs the HP regions to be void.

The ISOXML file exist in the current `development` branch (https://github.com/watts-dev/watts/blob/development/examples/example6_adv_multiapps/unitcell_nogap_hom_xml_G11_df_MP.xml)

- [__watts_exec.py__](watts_exec.py): Workflow definition. This is the file to execute to run the problem described above.
- [__unitcell_nogap_hom_xml_G11_df_MP.xml__](unitcell_nogap_hom_xml_G11_df_MP.xml): ISOXML file containing multigroup XS for Griffin

> - [__3D_unit_cell_FY21_level-1_bison.e__](3D_unit_cell_FY21_level-1_bison.e): Main mesh file.

Reviewer: ```suggestion
- [__3D_unit_cell_FY21_level-1_bison.e__](3D_unit_cell_FY21_level-1_bison.e): Mesh file with heat pipe holes for BISON/MOOSE subapplication.
```

> - [__3D_unit_cell_FY21_supersimple.e__](3D_unit_cell_FY21_supersimple.e): Another mesh file??

Reviewer: ```suggestion
- [__3D_unit_cell_FY21_supersimple.e__](3D_unit_cell_FY21_supersimple.e): Main mesh file used by Griffin parent app.
```
